plugins/org.eclipse.n4js.ide/src/org/eclipse/n4js/ide/editor/contentassist/CustomN4JSParser.java

```diff
@@ -89,6 +96,39 @@ public RecognizerSharedState getState() {
 			return state;
 		}
 
+		@Override
+		public void announceConsume() {
+			if (marked <= 0) {
+				localParamStack.clear();
+			}
+			super.announceConsume();
+		}
+
+		/*
+		 * This is here to provoke a compile error against a version of Xtext that has a fix for this.
+		 */
+		private List<Integer> getLocalParamStack() {
```

Comment:
So we will get a compile error here when updating Xtext? Good idea!

Please change `has a fix for this.` to `has a fix for GH-1756.`.